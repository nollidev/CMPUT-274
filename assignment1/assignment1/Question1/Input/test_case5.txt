harry.potter@gmail.com frodo.baggins@yahoo.com tony.stark@gmail.com princess.leia@gmail.com jack.dawson@yahoo.com forrest.gump@outlook.com marty.mcfly@gmail.com john.wick@yahoo.com indiana.jones@gmail.com dorothy.gale@outlook.com darth.vader@gmail.com elsa.icequeen@yahoo.com rocky.balboa@gmail.com nemo@yahoo.com joker@gmail.com bruce.wayne@outlook.com hannibal.lecter@gmail.com jason.bourne@yahoo.com katniss.everdeen@gmail.com rick.deckard@outlook.com