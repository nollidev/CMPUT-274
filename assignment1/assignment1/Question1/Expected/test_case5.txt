Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
Valid
